src/App.js

```diff
@@ -1,25 +1,81 @@
-import logo from './logo.svg';
-import './App.css';
+import SearchPage from "../src/components/SearchPage";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+export default function App() {
+  const pageName = "Suche";
+  const concerts = [
```

Comment:
Just personal preference: to get a better overview maybe use a .json file with concerts and import instead of having the list inside your App.js

Reply:
Good point! It was a bit difficult to get a nice .json list here so i temporarily used this way. But right! I could´ve out them all in a .json file :) I actually solved this problem (for the moment) by scraping the data of a concert Website :)